lamby/lamby.py

```diff
@@ -44,11 +50,112 @@ def uninit():
 @click.argument('files', nargs=-1)
 @click.option('-m', '--message')
 def commit(files, message):
-    click.echo('Commiting following files:')
+    lamby_dir = './.lamby'
+    if not os.path.isdir(lamby_dir):
+        click.echo('Lamby project has not been initialized in ' + os.getcwd())
+        sys.exit(1)
+
+    if len(files) == 0:
+        files = search_file_type('.', 'onnx')
+
+    log = deserialize_log()
+
     for file in files:
-        click.echo('\t' + file)
-    if message is not None:
-        click.echo('Commit Message: ' + message)
+
+        basename = os.path.basename(file)
+
+        if not os.path.isfile(file):
+            click.echo(file + ' is not a file')
+            continue
+
+        if file.split('.')[-1] != 'onnx':
+            click.echo(file + ' is not an onnx file')
+            continue
+
```

Comment:
These two should be done as input validation before we start committing. Not sure how i feel about using lamby commit and having 1 or 2 fail cuz of wrong input and the rest go through. I would rather have it validate input and fail them all together.

Reply:
agreed will fix